modules/ROOT/pages/6.0-release-notes-core-changes.adoc

```diff
@@ -0,0 +1,296 @@
+= TinyMCE 6.0: the core changes
+:navtitle: Core changes
+:description: The core changes for TinyMCE 6.0
+:keywords: releasenotes, bugfixes, defaults
+
+This section documents the three core changes made to {productname} 6.0.
+
+This section also includes four tables that set out the:
+
+* renamed APIs, Commands, Plugins, Options, and UI & UX elements
+* values changed in a default {productname} 6.0 instance.
+* previously deprecated and now removed items.
+* never documented, and consequently never supported, items that have now been removed.
+
+
+* xref:core-changes[Core changes]
+* xref:things-we-renamed[Things we renamed]
+* xref:default-value-changes[Default value changes]
+* xref:previously-deprecated-items-now-removed[Previously deprecated items now removed]
+* xref:previously-undocumented-items-removed-without-prior-deprecation[Previously undocumented items removed without prior deprecation]
+
+
+// tag::core-changes[]
+[[core-changes]]
+== Core changes
+
+. {productname} 6.0 now requires a browser capable of running https://tc39.es/ecma262/2020/[ECMAScript 2020] (ES2020).
++
+NOTE: In particular, this means {productname} 6.0 does not support Microsoft Internet Explorer 11.
++
+. Premium Plugins released with {productname} 6.0 are not backwards compatible with earlier versions of {productname}.
+. {productname} 6.0 is now available in three further language:
+* Hindi (hi);
+* Malay (ms); and
+* Vietnamese (vi)
+
+// end::core-changes[]
+
+
+// tag::things-we-renamed[]
+[[things-we-renamed]]
+== Things we renamed
+
+[cols="1,1,1,1,1"]
+|===
+| Old name                            | New name                         | Element                             | Notes                                                                                                                                                                                                                                                                                                         | for more information
```

Comment:
```suggestion
| Old name                            | New name                         | Element                             | Notes                                                                                                                                                                                                                                                                                                         | For more information
```
Sentence case? Did also wonder if "More information" would render a bit nicer?

Reply:
> For more information

This has been corrected.

> Did also wonder if “More information” would render a bit nicer?
> 
> [and] if “see” should be capitalised?

Good questions.

The extant text encourages reading the header as an initial clause with each entry in a cell below as a followng dependent clause.

In effect, a series of complete sentences:

_For more information see [thing to see]._

Which is also why _see_ is not capitalised: each time, the word is the beginning of a dependent clause, not the beginning of a new sentence.

If this isn’t the reading that occurs to you, your point about ‘rendering’ above may be the key. If the headers aren’t cleanly readable (because of forced line-breaks, say), the reading intent may be lost.

Reply:
That makes sense. I'm happy with it as it is now